src/app/public/modules/description-list/description-list.component.ts

```diff
@@ -48,7 +48,10 @@ import {
   selector: 'sky-description-list',
   templateUrl: './description-list.component.html',
   styleUrls: ['./description-list.component.scss'],
-  providers: [SkyDescriptionListService],
+  providers: [
+    SkyDescriptionListAdapterService,
```

Comment:
Now that the adapter is provided by the component, we no longer need to generate a renderer using the `Renderer2Factory`.
Instead of this:
```
private renderer: Renderer2;

constructor(
  private rendererFactory: RendererFactory2
) {
  this.renderer = this.rendererFactory.createRenderer(undefined, undefined);
}
```

We can now do this:
```
constructor(
  private renderer: Renderer2
) {}
```